src/extensive_list_of_resrouce.md

```diff
@@ -0,0 +1,247 @@
+# Computer Graphics Programming Resources
+
+List of freely available resources to study computer graphics programming. Composed by [u/Better_Pirate_7823](https://www.reddit.com/user/Better_Pirate_7823/) 
```

Comment:
If this comes from an existing reddit post, credit the original, credit the author who converted and edited it to this format